easybuild/easyconfigs/a/admixture/admixture-1.3.0.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = "Tarball"
+
+name = 'admixture'
+version = '1.3.0'
+
+homepage = 'https://dalexander.github.io/admixture/index.html'
+description = """ Software tool for maximum likelihood estimation of individual
+ancestries from multilocus SNP genotype datasets. It uses the same statistical
+model as STRUCTURE but calculates estimates much more rapidly using a
+fast numerical optimization algorithm. """
+
+toolchain = SYSTEM
+
+source_urls = ['https://dalexander.github.io/admixture/binaries/']
+sources = ['https://dalexander.github.io/admixture/binaries/admixture_linux-1.3.0.tar.gz']
```

Comment:
the `sources` should be just filenames, and not include the URL - that's what `source_urls` is for
```suggestion
sources = [f'{name}_linux-{version}.tar.gz']
```